src/template.md

```diff
@@ -0,0 +1,30 @@
+# {{title}} (`{{type}}`)
```

Comment:
This looks good. I think we could make a few tweaks e.g.

* examples could just be a bulleted list.
* i'd move type out of title for item (that way anchor links will be nice)

However, these are minor tweaks we can quickly adjust later.